server/app.js

```diff
@@ -1,82 +1,89 @@
-const colors = require("colors");
-const path = require("path");
-const http = require("http");
-const express = require("express");
-const socketio = require("socket.io");
-const { notFound, errorHandler } = require("./middleware/error");
-const connectDB = require("./db");
-const { join } = require("path");
-const cookieParser = require("cookie-parser");
-const logger = require("morgan");
+const colors = require('colors');
+const path = require('path');
+const http = require('http');
+const express = require('express');
+const socketio = require('socket.io');
+const { notFound, errorHandler } = require('./middleware/error');
+const connectDB = require('./db');
+const { join } = require('path');
+const cookieParser = require('cookie-parser');
+const logger = require('morgan');
 const passport = require('passport');
 const session = require('express-session');
 require('./utils/oauthGoogleStrategy');
 
-const authRouter = require("./routes/auth");
-const userRouter = require("./routes/user");
-const oauthRouter = require("./routes/oauth");
-const appointmentRouter = require("./routes/appointment");
-const meetingRouter = require("./routes/meeting");
-
+const authRouter = require('./routes/auth');
+const userRouter = require('./routes/user');
+const oauthRouter = require('./routes/oauth');
+const appointmentRouter = require('./routes/appointment');
+const meetingRouter = require('./routes/meeting');
 
 const { json, urlencoded } = express;
 
 connectDB();
 const app = express();
 const server = http.createServer(app);
 
+// Initializes our cloudinary
+cloudinary.config({
```

Comment:
Setting up involves some more elements, you can see this example [https://cloudinary.com/blog/node_js_file_upload_to_a_local_server_or_to_the_cloud](example)

Reply:
I think this set up should be fine, would you be able to point out what is currently be missing? 

I assume @garzeah followed this guide instead: https://cloudinary.com/documentation/node_integration

Reply:
@bonnieli Yeah.. this is set up should be fine. Please resolve the conflicts though :)

Reply:
Resolved, thanks!